src/sms/adapters/inbound/fastapi_/routes.py

```diff
@@ -38,6 +37,30 @@
     description="The database and collection to query. Format: db_name.collection",
 )
 
+query_params_desc = (
+    "Query parameters used to specify the documents affected by the request."
+)
+query_params_open_api: dict[str, Any] = {
+    "parameters": [
+        {
+            "description": query_params_desc,
+            "in": "query",
+            "name": "Criteria",
+            "required": False,
+            "schema": {
+                "description": query_params_desc,
+                "title": "Criteria",
+                "type": "object",
```

Comment:
I think you still need to add `"additionalProperties": True` here.

Reply:
Added!